runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -368,6 +363,13 @@ public void initializeState(StateInitializationContext context) throws Exception
 
       timerInternals = new FlinkTimerInternals();
     }
+
+    outputManager = createOutputManager();
```

Comment:
Perhaps add a comment why we need it?

Reply:
Left a comment in the new lock retrieval method.